packages/gatsby-core-utils/src/create-require-from-path.ts

```diff
@@ -0,0 +1,22 @@
+import Module from "module"
```

Comment:
`"allowSyntheticDefaultImports": true` is needed for this to work with `import * as Module from "module"`